Source/Scene/Cesium3DTileset.js

```diff
@@ -1745,52 +1737,27 @@ define([
                 stack.push(children[i]);
             }
             if (tile !== root) {
-                unloadTileFromCache(tileset, tile);
-                tile.destroy();
+                destroyTile(tileset, tile);
                 --statistics.numberOfTilesTotal;
             }
         }
         root.children = [];
     }
 
-    function unloadTileFromCache(tileset, tile) {
-        var node = tile.replacementNode;
-        if (!defined(node)) {
-            return;
-        }
-
-        var statistics = tileset._statistics;
-        var replacementList = tileset._replacementList;
-        var tileUnload = tileset.tileUnload;
+    function unloadTile(tileset, tile) {
+        tileset.tileUnload.raiseEvent(tile);
+        tileset._statistics.decrementLoadCounts(tile.content);
+        --tileset._statistics.numberOfTilesWithContentReady;
+        tile.unloadContent();
+    }
 
-        tileUnload.raiseEvent(tile);
-        replacementList.remove(node);
-        statistics.decrementLoadCounts(tile.content);
-        --statistics.numberOfTilesWithContentReady;
+    function destroyTile(tileset, tile) {
+        tileset._cache.unloadTile(tileset, tile, unloadTile);
+        tile.destroy();
     }
 
     function unloadTiles(tileset) {
-        var trimTiles = tileset._trimTiles;
-        tileset._trimTiles = false;
-
-        var replacementList = tileset._replacementList;
-
-        var totalMemoryUsageInBytes = tileset.totalMemoryUsageInBytes;
-        var maximumMemoryUsageInBytes = tileset._maximumMemoryUsage * 1024 * 1024;
-
-        // Traverse the list only to the sentinel since tiles/nodes to the
-        // right of the sentinel were used this frame.
-        //
-        // The sub-list to the left of the sentinel is ordered from LRU to MRU.
-        var sentinel = tileset._replacementSentinel;
-        var node = replacementList.head;
-        while ((node !== sentinel) && ((totalMemoryUsageInBytes > maximumMemoryUsageInBytes) || trimTiles)) {
-            var tile = node.item;
-            node = node.next;
-            unloadTileFromCache(tileset, tile);
-            tile.unloadContent();
-            totalMemoryUsageInBytes = tileset.totalMemoryUsageInBytes;
-        }
+        tileset._cache.unloadTiles(tileset, unloadTile);
```

Comment:
Should this just be inlined now that it's a one-liner and it's only called from `update`?

Reply:
I kind of like how each operation on tiles has a specific name to it, e.g. `processTiles`, `selectTiles`, `requestTiles`, `updateTiles`, `unloadTiles`. Just a style preference.